app/account/page.tsx

```diff
@@ -38,7 +38,9 @@ export default function AccountPage() {
   return (
     <>
       {loading || !data?.usersCollection ? (
-        <Loading />
+        <Flex minH="100vh" align="center" justify="center">
+          <Loading />
+        </Flex>
```

Comment:
I added min height so when the page is loading, the spinner is located in the middle. 
<table>
  <tr>
    <td valign="top"><img width="200" alt="Screenshot 1" src="https://github.com/tabi-memo/frontend/assets/70562492/c7437af8-f3cd-4424-be91-21a2156a1e1b"></td>
    <td valign="top"><img width="200" alt="Screenshot 2" src="https://github.com/tabi-memo/frontend/assets/70562492/38b143ea-fae0-4316-8362-d8bde55f138c"></td>
  </tr>
</table>